src/renderer/components/layout/sidebar.tsx

```diff
@@ -71,12 +71,19 @@ export class Sidebar extends React.Component<Props> {
     });
   }
 
+  @autobind()
+  onDoubleClick() {
+    if (!this.props.isPinned) {
+      this.props.toggle()
+    }
+  }
+
```

Comment:
I we're leaving double click feature, then it should work in both ways?

Reply:
Now with further consideration, I think it would just be best not to have this in yet.